openpathsampling/pathmover.py

```diff
@@ -1503,12 +1502,6 @@ class RandomAllowedChoiceMover(RandomChoiceMover):
     from sub movers that actually can succeed because they have samples in all
     required input_ensembles
 
-    Attributes
-    ----------
-    movers : list of PathMover
-        the PathMovers to choose from
-    weights : list of floats
-        the relative weight of each PathMover (does not need to be normalized)
```

Comment:
for now, I'm fine to either remove or leave, whichever you prefer. Might be better to put them back in, though (you'll see the changes easily in this PR's diff), so that these docstrings are in place for Traunkirchen.